IoTHubManager/DeploymentsTests.cs

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using Helpers.Http;
+using System;
+using System.Net;
+using IoTHubManager.Models;
+using Xunit;
+using Newtonsoft.Json;
+
+namespace IoTHubManager
+{
+    [Collection("IoTHub Manager Tests")]
+    public class DeploymentsTests
+    {
+        private const string TEST_PACKAGE_JSON =
```

Comment:
Can you copy this into Constants file or some common file. I see this is also being used in packageTest

Reply:
Done but updating in https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet-tests/pull/42 because I pressed squash & merge.